src/acp/matcher.test.ts

```diff
@@ -141,14 +140,20 @@ const MOCK_CLIENT_WEBID_1 = DataFactory.namedNode(
 const MOCK_CLIENT_WEBID_2 = DataFactory.namedNode(
   "https://your.app/registration#it"
 );
+const MOCK_CLIENT_WEBID_3 = DataFactory.namedNode("test_string_client_id");
+const MOCK_CLIENT_WEBID_4 = DataFactory.namedNode("other_string_client_id");
```

Comment:
Also, isn't a `namedNode` always an IRI? Wouldn't it be possible to this to be just a plain string?